policyengine_canada/tests/gov/provinces/on/tax/income/credits/ostc/on_sales_tax_credit_reduction.yaml

```
@@ -0,0 +1,31 @@
+- name: Single in Onatrio with income over reduction threshold
```

And elsewhere in file
```suggestion
- name: Single in Ontario with income over reduction threshold
```